### PR TITLE
Clarify stabilty guarantees; drop 'proof of concept' and 'experimental'.

### PR DESCRIPTION
**Pull request by milessabin**

### CODE REVIEW

File: docs/src/site/index.md [hunk 1]
```diff
@@ -3,16 +3,15 @@ layout: default
...
-  contributions are welcomed as we look to improve the project. This
-  project is evolving quickly and we currently make no guarantees about what
-  might drastically change in the near future.</p> </div>
+Cats is a library intended to provide abstractions for functional
+programming in the [Scala programming language](https://scala-lang.org).
+The name is a playful shortening of the word *category*.
+
+<div class="msg warn"> <p><strong> Cats is a new project under active
+  development</strong>. Feedback and contributions are welcomed as we look
+  to improve the it. This project is evolving quickly and we are making no
```
Comment (DavidGregory084) [inline docs/src/site/index.md]: Typo "the it"

**Comment by mikejcurry** [comment]
LGTM - explicit targets are good!

File: docs/src/site/index.md [hunk 2]
```diff
@@ -3,16 +3,15 @@ layout: default
...
-functional programming in the
-[Scala programming language](https://scala-lang.org). The name is a
-playful shortening of the word *category*.
-
-<div class="msg warn"> <p><strong> Cats is currently an experimental
-  project under active development</strong>. Feedback and
-  contributions are welcomed as we look to improve the project. This
-  project is evolving quickly and we currently make no guarantees about what
-  might drastically change in the near future.</p> </div>
+Cats is a library intended to provide abstractions for functional
```
Comment (InTheNow) [inline docs/src/site/index.md]: Does it "intend to " or does it achieve it? ie:

> Cats is a library that provides abstractions for functional.....

Comment (milessabin) [inline docs/src/site/index.md]: Yes, that's better.

**Comment by ceedubs** [comment]
:+1:

**Comment by adelbertc** [comment]
:+1:

**Pull request merged**